packages/server/lib/util/print-run.ts

```diff
@@ -508,13 +508,16 @@ export function displayVideoProcessingProgress (opts: { videoName: string, video
         table.push([
           gray('-'),
           gray('Finished processing:'),
-            `${formatPath(opts.videoName, getWidth(table, 2), 'cyan')}`,
-            gray(dur),
+          gray(dur),
         ])
 
         console.log(table.toString())
 
         console.log('')
```

Comment:
Not sure we need a new line between the video details and the video output name.